### PR TITLE
Add a pop up dialog box for Polls/Survey done ! #702 🌟🌟🌟

### PR DESCRIPTION
**Pull request by Subhajit-2023-44**
Hello, PR
Add new feature - #702
Add a pop up dialog box for Polls/Survey done properly -

https://github.com/user-attachments/assets/cf0b73a1-6433-415e-bc69-124ca1930e40

Screenshot 2024-10-31 105052
Screenshot 2024-10-31 105106

## Type of PR

- [ ] Bug fix
- [X] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): ___________  

## Checklist

- [X] I have performed a self-review of my code.
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

Thank you for reviewing my pull request!

### CODE REVIEW

**Comment by Subhajit-2023-44** [comment]
@swaraj-das @AnitSarkar123 done, please merge my PR with level 3..

I add a pop up dialog box for Polls/Survey (if you vote once, you won't see it again after reload..) #702

**Comment by AnitSarkar123** [comment]
> @swaraj-das @AnitSarkar123 done, please merge my PR with level 3..
> 
> I add a pop up dialog box for Polls/Survey (if you vote once, you won't see it again after reload..) #702

Great 👍 @swaraj-das

**Comment by AnitSarkar123** [comment]
> > @swaraj-das @AnitSarkar123 done, please merge my PR with level 3..
> > 
> > I add a pop up dialog box for Polls/Survey (if you vote once, you won't see it again after reload..) #702
> 
> Great 👍 @Subhajit-2023-44 

@swaraj-das Marge it

**Comment by Subhajit-2023-44** [comment]
> > @swaraj-das @AnitSarkar123 done, please merge my PR with level 3..
> > I add a pop up dialog box for Polls/Survey (if you vote once, you won't see it again after reload..) #702
> 
> Great 👍 @Subhajit-2023-44

Thank You 🙏🏻

**Comment by Subhajit-2023-44** [comment]
@swaraj-das please merge my PR with Level 3

Thank You

File: index.html [hunk 1]
```diff
@@ -754,6 +754,210 @@ <h2>Feedback Form</h2>
 
     </script>
     </div>
+
```
[the remark this answers is not in the document]
Comment (Subhajit-2023-44) [reply index.html:757]: @swaraj-das okay wait I will give a new PR soon,

But you never told me 😓😓

**Pull request closed without merging**